### PR TITLE
Created the ExpenseLogAggregate table with associated migrations

### PR DESCRIPTION
**Pull request by guptamo**

### CODE REVIEW

File: hr/api/models.py [hunk 1]
```diff
@@ -4,3 +4,8 @@ class ExpenseEventLog(models.Model):
     event_data = models.TextField()
     sequence = models.IntegerField()
     guid = models.IntegerField()
+
+class ExpenseLogAggregate(models.Model):
+    amount = models.DecimalField(max_digits = 7, decimal_places = 2)
```
Comment (can-z) [inline hr/api/models.py]: we should allow more digits as having max_digits=7 would only allow 5 digits before the decimal points.

File: hr/api/models.py [hunk 2]
```diff
@@ -4,3 +4,8 @@ class ExpenseEventLog(models.Model):
...
+    description = models.CharField(max_length = 200)
```
Comment (can-z) [inline hr/api/models.py]: minor python styling issue: usually there's no spaces around equal signs in kwargs.

Comment (guptamo) [inline hr/api/models.py]: removed the spaces

File: hr/api/models.py (hunk 1 above)
Comment (guptamo) [inline hr/api/models.py]: added digits to the amount model field

**Pull request merged**